src/plugins/azgenerator/TemplateAzureCliCustom.ts

```diff
@@ -128,61 +134,7 @@ function GetCommandBody(model: CodeModelAz, required: boolean, needUpdate: boole
-            do {
-                if (model.MethodParameter_IsFlattened) {
-                    let bodyName = model.MethodParameter_Name;
-                    output_body.push("    " + bodyName + " = {}");
-                    let body = model.MethodParameter;
-
-                    while (model.SelectNextMethodParameter()) {
-                        let access = "    " + bodyName;
-                        let param = model.MethodParameter;
-                        let oriParam = (param['originalParameter']);
-                        if (oriParam == body) {
-                            if (param['pathToProperty']?.length == 1) {
-                                let pathParam = param['pathToProperty'][0];
-                                access += `.setdefault('${pathParam.language['python'].name}', {})`;
-                                access += `['${model.MethodParameter_Name}'] = `;
-                            } else {
-                                access += `['${model.MethodParameter_Name}'] = `;
-                            }
-                            if (model.MethodParameter_IsList) {
-                                if (model.MethodParameter_Type != SchemaType.Dictionary) {
-                                    // a comma separated list
-                                    access += "None if " + model.MethodParameter_MapsTo + " is None else " + model.MethodParameter_MapsTo;
-                                }
-                                else {
-                                    // already preprocessed by actions
-                                    access += model.MethodParameter_MapsTo;
-                                }
-                            }
-                            else if (model.MethodParameter_Type != SchemaType.Dictionary) {
-                                access += model.MethodParameter_MapsTo + "  # " + model.MethodParameter_Type; // # JSON.stringify(element);
-                            }
-                            else {
-                                access += "json.loads(" + model.MethodParameter_MapsTo + ") if isinstance(" + model.MethodParameter_MapsTo + ", str) else " + model.MethodParameter_MapsTo
-                                required['json'] = true;
-                            }
-
-                            if (isUpdate) {
-                                output_body.push("    if " + model.MethodParameter_MapsTo + " is not None:");
-                                output_body.push("    " + access);
-                            }
-                            else {
-                                output_body.push(access);
-                            }
-
-                        } else {
-                            break;
-                        }
-                    }
-                }
-
-            }
-            while (model.SelectNextMethodParameter());
-
-        }
+        // with x-ms-client-flatten it doesn't need this part now 
```

Comment:
what if there's no "x-ms-client-flatten" in swagger?

Reply:
we will need this parameter anyhow since `payload-flattening-threshold` is unrecommended. the problem is where do we put this parameter. As @RodgeFu  suggested. we can try to make it work with directive without change the original swagger. but will need some experiments on that.

Reply:
we could have if there, if "x-ms-client-flatten" is there you skip the code, otherwise old code should stay